backend/minors/ART.yaml

```diff
@@ -0,0 +1,100 @@
+  literary, economic, scientific, technological or religious culture of a 
+  particular period, community or region. Art historians frequently collaborate 
+  with scholars in the sciences as they study the material composition of 
+  artworks, and the field is conducive to technology-based projects and methods 
+  (such as imaging technologies and data visualization).
+
+  "The field is by nature interdisciplinary:" it draws on the ideas, information, 
+  theories and resources of diverse branches of knowledge, as art historians 
+  seek to understand the cultural fabric of which a given work of art forms a 
+  part. As such, a minor in the History of Art may complement a major in a wide 
+  range of disciplines across all University divisions. The skills of 
+  observation, analysis and research that students cultivate through 
+  art-historical practice prove valuable in fields from medicine and law to 
+  engineering and biological- or conservation science, along with all branches 
+  of the humanities and social sciences.
+
+urls:
+- https://artandarchaeology.princeton.edu/about-us/undergraduate#minorinarthistory
+contacts:
+- type: Director of Undergraduate Studies
+  name: Basile Baudez
+  email: bbaudez@princeton.edu
+req_list: 
+- name: Distribution Requirements
+  max_counted: 1
+  min_needed: 5
+  explanation: Minors may consult with Art & Archaeology’s Director of 
+  Undergraduate Studies about their course planning, but are not required 
+  to do so. Five ART courses; none to be taken P/D/F.
+  double_counting_allowed: false
+  max_common_with_major: 1
+  pdfs_allowed: 0
+  dept_list:
+  - ART
+  req_list:
+  - name: Introductory
+    max_counted: 5
+    min_needed: 1
+    explanation: All minors must take ART100, the Department’s introductory course
+    course_list: 
+    - ART100
+  - name: 200 level
+    max_counted: 5
+    min_needed: 1
+    explanation: One of the five courses must be at 200-level
+    course_list: 
+    - ART 2*
+  - name: 400 level
+    max_counted: 5
+    min_needed: 1
+    explanation: One of the five courses must be at 400-level
+    course_list: 
+    - ART 3*
+- name: Restrictions
+  max_counted: 0
+  min_needed: 1
+  explanation: |-
+    Minors must take one course in each of the department’s three 
+    distribution areas: Ancient (Group 1), Medieval and Early Modern 
+    (Group 2), Modern (Group 3). This distribution requirement may be met 
+    through courses at any level, and in any geographic area. Group 
+    designations are included in the Registrar’s course listings, and will 
+    also be communicated to students through a department announcement when 
+    course registration opens.
+
+    With the exception of cross-listed courses taught by core Art & 
+    Archaeology faculty, no cross-lists or cognates will count toward 
+    the minor (i.e., unless taught by core faculty, all courses must carry 
+    ART as the first or only subject code).
```

Comment:
Let's add a `no_req` requirement and an editor's note in the explanation. For an example, check out [URB.yaml](https://github.com/HoagieClub/plan/blob/3f6b6d2f530c825d6cbc8c43d103d1001a5ef9ba/backend/certificates/URB.yaml#L340-L355)